src/crs/__init__.py

```diff
@@ -36,6 +38,16 @@
             )
         },
         equality_delta=0.95,
+        qgis_template_srid=3857,
```

Comment:
It's PostGIS SRID. Remove it from definition, get if from existing method dynamically.